bin/test

```diff
@@ -58,6 +29,61 @@ var testsWorkspace = process.env.CODE_TESTS_WORKSPACE || testsFolder;
 console.log('### VS Code Extension Test Run ###');
 console.log('Current working directory: ' + process.cwd());
 
+function getDownloadUrl() {
+    if (process.env.CODE_DOWNLOAD_URL) {
+        return Promise.resolve(process.env.CODE_DOWNLOAD_URL);
+    }
+
+    return ghGot('repos/Microsoft/vscode/tags')
+        .then(function(result) {
+            // Filter out invalid tags and prerelease tags
+            var result = result.body.filter(function(tag) {
+                var parsed = semver.parse(tag.name);
+                return parsed && parsed.prerelease.length === 0;
+            });
```

Comment:
It's correct that I can't install an insiders release right? I tried it with `0.10.7-insiders`, but it didn't work so that's why I'm filtering them here.